changelogs/feature/new_adapter_structure.json

```diff
@@ -0,0 +1,5 @@
+{
+  "author": "Nemikor",
+  "pullrequestId": "106",
+  "message": "Replace archetype structure with new simplified one."
```

Comment:
This is very important change, we should be as precise as possible, hints:

- It's a new adapter structure, not archetype. 
- Switching from multi module adapters to no modules application structure